cli/Valet/PhpFpm.php

```diff
@@ -53,8 +53,8 @@ function install()
     function uninstall()
     {
         $this->brew->uninstallAllPhpVersions();
-        rename('/usr/local/etc/php', '/usr/local/etc/php-valet-bak'.time());
-        $this->cli->run('rm -rf /usr/local/var/log/php-fpm.log');
+        rename(BREW_PREFIX.'/etc/php', BREW_PREFIX.'/etc/php-valet-bak'.time());
+        $this->cli->run('rm -rf '.BREW_PREFIX.'/var/log/php-fpm.log');
```

Comment:
@soilSpoon I'm thinking maybe line 41 above also have the BREW_PREFIX on it too...?

Reply:
@drbyte Would it be better to go to the BREW_PREFIX?
I didn't modify the path because it is a log directory that has nothing to do with BREW.

Reply:
@soilSpoon In your default php.ini where is it defaulting for storing PHP logs when BREW_PREFIX is not /usr/local ?

eg:
```
; Error log file
; If it's set to "syslog", log is sent to syslogd instead of being written
; into a local file.
; Note: the default prefix is /usr/local/var
; Default Value: log/php-fpm.log
;error_log = log/php-fpm.log
```

Reply:
It is set to "syslog".

Reply:
> It is set to "syslog".

I see that with PHP 8. Probably is Homebrew's new standard (or is PHP's standard and brew is now leaving it alone).

maybe we should point this log file to the user's home directory, 
eg: `~/.config/valet/Log/php-fpm.log`

(We already point `~/.config/valet/Log/nginx-error.log` there)

Reply:
@drbyte Only need to modify the path of log files? Or is there another path that needs to be modified?

Reply:
That's the only remaining /usr/local/ thing AFAIK